tsconfig.json

```diff
@@ -0,0 +1,74 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                         /* Enable incremental compilation */
+    "target": "es5",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ES2021', or 'ESNEXT'. */
```

Comment:
Could we target `ES2020` or `ESNEXT` ?

Reply:
Set it to `ESNEXT` with https://github.com/Major-Eth/yearn-web-lib/pull/8/commits/dba3032a8f09d5d48d3759d79ff43511368db906